blockchain/v2/reactor_test.go

```diff
@@ -9,11 +9,12 @@ import (
 	"testing"
 	"time"
 
-	dbm "github.com/cometbft/cometbft-db"
 	"github.com/gogo/protobuf/proto"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	dbm "github.com/cometbft/cometbft-db"
+
```

Comment:
[observation] `make format` runs:

```
	find . -name '*.go' -type f -not -path "*.git*" -not -name '*.pb.go' -not -name '*pb_test.go' | xargs gofmt -w -s
	find . -name '*.go' -type f -not -path "*.git*"  -not -name '*.pb.go' -not -name '*pb_test.go' | xargs goimports -w -local github.com/cometbft/cometbft
```

even though golangci.yml doesn't have any rules to special case the order of imports for `github.com/cometbft/cometbft`. In other words, this PR seems fine to me but a future contributor can accidentally modify the order of imports and we won't get signal on it.

IMO the order of imports isn't important so I propose we modify the Makefile to do this:

```
fmt:
	@echo "--> Running golangci-lint --fix"
	@golangci-lint run --fix
```

and then re-run `make format` to fix the issues that golangci.yml has identified and can fix.

Reply:
[ac21986](https://github.com/celestiaorg/celestia-core/pull/1516/commits/ac21986f307aaea60fb6c9f1cdd7b3166394374e)